src/Olry/MentalNoteBundle/Resources/views/Entry/line.html.twig

```diff
@@ -8,6 +8,11 @@
         <div class="col-sm-12 col-md-8">
             {% if show_actions|default(false) %}
             <div class="pull-right">
+
+                <a class="twitter-share-button" href="https://twitter.com/intent/tweet?text={{ entry.url }}" target="_blank"><i class="fa fa-twitter-square" aria-hidden="true"></i></a>
```

Comment:
* that class is probably from twitter's example code, no necessary here, is it?
* title missing